src/driver/driver.cpp

```diff
@@ -156,7 +156,9 @@ void EvolutionDriver::InitializeBlockTimeSteps() {
 void EvolutionDriver::SetGlobalTimeStep() {
   // don't allow dt to grow by more than 2x
   // consider making this configurable in the input
-  tm.dt *= 2.0;
+  if (tm.dt < 0.1 * std::numeric_limits<Real>::max()) {
+    tm.dt *= 2.0;
+  }
```

Comment:
Wow... you hit this? I'm impressed.

Reply:
Tracking down another problem I used 
```
#include <fenv.h> 
feenableexcept(FE_DIVBYZERO | FE_INVALID | FE_OVERFLOW);
```
and immediately found this one directly during init

Reply:
heh nice